jetson/percep/artag_detector.hpp

```diff
@@ -24,4 +26,5 @@ class TagDetector {
     Point2f getAverageTagCoordinateFromCorners(const vector<Point2f> &corners);     //takes detected AR tag and finds center coordinate for use with ZED
     pair<Tag, Tag> findARTags(Mat &src, Mat &depth_src, Mat &rgb);                  //detects AR tags in a given Mat
     double getAngle(float xPixel, float wPixel);                                    //finds the angle from center given pixel coordinates
+    void tagfound(rover_msgs::Target* arTags, pair<Tag, Tag> &tagPair,int tag, int buffer, Mat &depth_img, Mat &src); //if AR tag found, updates distance, bearing, and id
```

Comment:
Please move the comment to the line above, I can't even read the comment without scrolling to the side at which point I can't read the function stub.

The function name doesn't seem to really match the comment. In other words, when I read the function name, it does not correctly imply what the function does. Also, function names should start with a verb like the functions above it.